### ISSUE TITLE
[Bug] Discord Login Does Not Work

### ISSUE THREAD

**Issue by Kyusetzu** [toasterofbread/spmp#128]
**Describe the bug**

Discord login not working

**Steps To Reproduce**

- Go to settings
- Go to discord
- Login via website
-> App crashes

- Login via Authorization
-> Error (see below)

**Expected behaviour**

Logging into Discord successfully 

**Environment**
- OS name and version (e.g. Android 13): 
Android 13
- SpMp version (include commit hash if you are using a non-release build): 
0.1.4
- Accounts (were you signed in to YouTube, Discord, etc. in the app?): 
YouTube
**Screenshots**

**Logs**

```
java.lang.IllegalArgumentException: At least one of guild_id and custom_images_channel_category_id required
	at com.toasterofbread.spmp.platform.DiscordStatus.<init>(Unknown Source:39)
	at com.toasterofbread.spmp.platform.DiscordStatus.<init>(Unknown Source:41)
	at com.toasterofbread.spmp.PlayerService.onDiscordAccountTokenChanged(Unknown Source:71)
	at com.toasterofbread.spmp.PlayerService.access$onDiscordAccountTokenChanged(Unknown Source:0)
	at com.toasterofbread.spmp.PlayerService$prefs_listener$1.onChanged(Unknown Source:24)
	at com.toasterofbread.spmp.platform.ProjectPreferences$Listener$DefaultImpls.onSharedPreferenceChanged(Unknown Source:16)
	at com.toasterofbread.spmp.PlayerService$prefs_listener$1.onSharedPreferenceChanged(Unknown Source:0)
	at android.app.SharedPreferencesImpl$EditorImpl.notifyListeners(SharedPreferencesImpl.java:637)
	at android.app.SharedPreferencesImpl$EditorImpl.apply(SharedPreferencesImpl.java:511)
	at com.toasterofbread.spmp.platform.ProjectPreferences.edit(Unknown Source:24)
	at com.toasterofbread.settings.model.SettingsValueState.save(Unknown Source:11)
	at com.toasterofbread.settings.model.SettingsValueState.setValue(Unknown Source:35)
	at com.toasterofbread.spmp.ui.layout.prefspage.DiscordLoginPageKt$getDiscordLoginPage$1$PageView$1.invoke(Unknown Source:24)
	at com.toasterofbread.spmp.ui.layout.prefspage.DiscordLoginPageKt$getDiscordLoginPage$1$PageView$1.invoke(Unknown Source:2)
	at com.toasterofbread.spmp.ui.layout.DiscordManualLoginKt$DiscordManualLogin$1$1.invokeSuspend(Unknown Source:116)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(Unknown Source:8)
	at kotlinx.coroutines.DispatchedTask.run(Unknown Source:100)
	at androidx.compose.ui.platform.AndroidUiDispatcher.access$performTrampolineDispatch(Unknown Source:22)
	at androidx.compose.ui.platform.AndroidUiDispatcher$dispatchCallback$1.run(Unknown Source:2)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:240)
	at android.os.Looper.loop(Looper.java:351)
	at android.app.ActivityThread.main(ActivityThread.java:8404)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:584)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1013)
	Suppressed: coil.network.HttpException: [androidx.compose.ui.platform.MotionDurationScaleImpl@eab0f2a, androidx.compose.runtime.BroadcastFrameClock@1f0711b, StandaloneCoroutine{Cancelling}@bb892b8, AndroidUiDispatcher@c9ab891]
```

**Comment by toasterofbread** [comment]
This might have been fixed by 34538c687294554f0de371f41e9514e63ad6835e, please try [the latest build](https://github.com/toasterofbread/spmp/actions/runs/6412011135) and see if it works.

**Comment by Kyusetzu** [comment]
Update: 34538c687294554f0de371f41e9514e63ad6835e sure fixed the issue. Thank you.
